### ISSUE TITLE
Instance: The fact that new reply messages have appeared is not announced to users | WCAG 2.2 Violation - There are changes in status in the interface that are not announced to users.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#48]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [4.1.3 Status Messages](https://www.w3.org/TR/WCAG22/#status-messages)

### Screenshot

![Screenshot showing a `.chat-new-message-indicator` element with the text 'New'. This element is displayed visually on screen, but the fact that a new reply has appeared is not actually announced to assistive technology users via live region.](https://github.com/user-attachments/assets/0d873ef1-68e9-4e47-9bc5-9c7c8b99fa4e)

### Code Reference

```
<span class="chat-new-message-indicator" style="display: inline-flex; visibility: visible; opacity: 1;">New</span>
```

#### Further information:

1. 

### Instances

1.
